### DISCUSSION TITLE
Clear content of the log element

### DISCUSSION THREAD

**Discussion by pythonormen** [zauberzeug/nicegui#516] [Q&A] [answered]
Is it possible to clear the content of the log element?

Tried the following but it does not work. :) 

Thank you

```py
from nicegui import ui

log = ui.log(max_lines=50).classes('w-full h-20')
ui.button('Clear log', on_click=lambda: log.set_value(None))
ui.button('Add to log', on_click=lambda: log.push("Hello"))

ui.run()
```

**Comment by falkoschindler** [comment]
That's a good question!
Currently it is not possible to clear the log. But we are already working on a solution: #414.

**Comment by falkoschindler** [comment, marked as answer]
With NiceGUI 1.2.0 you can simply call `log.clear()` to remove all lines. Thanks, @pythonormen for the initial idea for this feature!
